definitions/npm/winston_v3.x.x/flow_v0.34.x-/winston_v3.x.x.js

```diff
@@ -65,6 +65,7 @@ declare type $winstonConfigSubModule = {
 };
 
 declare type $winstonFormatSubModule = {
+  ((info: Object) => Object): () => $winstonFormat,
```

Comment:
`Object` is `any` from Flow 0.75 on. No way to type it more precisely? We often use `{}` or `{ +[string]: mixed }` depending on the context.

Reply:
In other places of this file an `Object` type is used, so I used it too to keep the module style.